src/org/zaproxy/zap/extension/exportreport/resources/help/contents/exportreport.html

```diff
@@ -1,334 +1,377 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"> 
 <HTML>
-	<HEAD>
-		<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
-		<TITLE>
-			Export Report
-		</TITLE>
-	</HEAD>
+    <HEAD>
+        <META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+        <TITLE>
+            Export Report
+        </TITLE>
+    </HEAD>
 
-	<BODY BGCOLOR="#ffffff">
+    <BODY BGCOLOR="#ffffff">
 
-		<H1><U>Export Report Version 1.0</U></H1>
-		Report Export module that allows users to customize content and export in a desired format.
-		<UL>
-			<LI>Created by <A href="https://github.com/JordanGS">JordanGS</A>.</LI>
-			<LI>Minimum Supported Version: Weekly Release ZAP_D-2016-08-15</LI>
-			<LI>Project on  <A href="https://github.com/JordanGS/workspace">GitHub</A>. You can also find a zip of the minimum supported release in the zap-download folder.</LI>
+        <H1><U>Export Report Version 2</U></H1>
+        Report Export module that allows users to customize content and export in a desired format.
+        <UL>
+            <LI>Created by <A href="https://github.com/JordanGS">JordanGS</A>.</LI>
+            <LI>Minimum Supported Version: Weekly Release ZAP_D-2016-09-05</LI>
+            <LI>Project on  <A href="https://github.com/JordanGS/workspace">GitHub</A>. You can also find a .zip of the minimum supported release in the zap-download folder.</LI>
+			<LI>Supported and incorporated in the <A href="https://wiki.jenkins-ci.org/display/JENKINS/ZAProxy+Plugin">Official OWASP Zed Attack Proxy Jenkins Plugin</A>.</LI>
 		</UL>
 
-		<HR>
-		<H2><U>Source Info</U></H2>
-		User entered and automatically retrieve data relevant to the report. Hover over each field in the extension for tool tip.
-		<UL>
-			<LI>Title - Defaults to Session Name</LI>
-			<LI>By</LI>
-			<LI>For</LI>
-			<LI>Scan Date - User entered date of AScan, defaults to current datetime</LI>
-			<LI>Report Date - Defaults to current datetime</LI>
-			<LI>Scan Version - Placeholder for future</LI>
-			<LI>Report Version - Defaults to current version of ZAP tool</LI>
-		</UL>
-		<H4>Version 1.0 UI</H4>
-		<UL style="list-style-type:none">
-			<LI>
-				<img src="images/SourceInfo.png" alt="Source Info">
-			</LI>
-		</UL>
-		<HR>
+        <H2><U>Table of Contents</U></H2>
+        <UL>
+			<LI>ZAP UI</LI>
```

Comment:
There are some places that are using tabs, while in the rest of the file is using spaces.